src/main/java/io/github/dftrakesh/cloverrest/model/updateInventory/Item.java

```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.updateInventory;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Item {
```

Comment:
already you created item class, so please remove it

Reply:
Deleted Item Class